examples/jsm/nodes/code/CodeNode.js

```diff
@@ -1,4 +1,4 @@
-import Node, { addNodeClass } from './Node.js';
+import Node, { addNodeClass } from '../core/Node.js';
 import { nodeProxy } from '../shadernode/ShaderNode.js';
```

Comment:
'../' here should be changed to '../../'.

Reply:
Why? doesn't seem to work that way

Reply:
I mistakenly thought for some reason that directory level of file has changed 🤦‍♂️

Reply:
No problems :)